curriculum/challenges/english/blocks/workshop-colorful-boxes/68ef0b3237fef991c92d9c19.md

```diff
@@ -0,0 +1,85 @@
+---
+id: 68ef0b3237fef991c92d9c19
+title: Step 3
+challengeType: 0
+dashedName: step-3
+---
+
+# --description--
+
+Now add a new `div` element with the class `flex-container` to your `header`. Then create six `div` elements with the class `box` inside the `div` you just created.
```

Comment:
you are asking to put it inside `header`, but it's not in the `header` in the next step. I am leaving comments to make it added below the header.


```suggestion
Now add a new `div` element with the class `flex-container` below your `header`. Then create six `div` elements with the class `box` inside the `div` you just created.
```